.gitignore

```diff
@@ -18,3 +18,11 @@ environment.sh
 
 # os files 
 .DS_Store
+
+
+.kitchen/**
+.kitchen/logs/kitchen.log
+terraform/modules/enclave/prometheus/.kitchen/
+*.plan
+*/.kitchen/**
+*.lock
```

Comment:
Looks like we should remove this line actually as per https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore